contributors.json

```diff
@@ -853,5 +853,9 @@
   "xxx32": {
     "country": "India",
     "name": "Aarushi"
+  },
```

Comment:
Let's order in ascending order by GitHub user name

Reply:
okay